docs/src/examples/02-flux-balance-analysis.jl

```diff
@@ -61,10 +61,11 @@ solution.objective
 
 solution.fluxes.PFK
 
+#md #!!! warning "Warning: Nested structures cannot be tabbed through yet"
+#md #    Julia currently does not evaluate nested structures like ConstraintTrees.
+#md #    Thus, you will not be able to tab through nested solutions, unless you save
+#md #    the intermediate result.
```

Comment:
this is a sudden negative. Do we say anywhere that the models an be tabbed? If so it would be better next to that (as a small limitation), no there (without the explicit limit, it looks like 1] Julia does not really support nested stuff, 2] whenever anything is nested, all tabbing will fail, and 3] no one knows what is the "immediate result" (does this also work with manually created sub-constraints?)

Recommendation: convert to positive hint: "!!!tip: You can use [tab] to quickly see what is in CTs. (Unfortunately due to typesystem limitations, this only works for the topmost level of the tree.)"

Reply:
(highly curated docwriting knowledge: negative information is never useful unless the danger is really dangerous)